js/newEddie.js

```diff
@@ -0,0 +1,15 @@
+var parents = document.getElementsByTagName("SELECT");
+var attr = ["selected", "disabled", "hidden"];
+var i;
+var j;
+
+for ( i=0; i<parents.length; i++){
+    
+    children = parents[i].firstElementChild;
```

Comment:
When we get to JS, we'll talk about global vs local variable scope and how use of global scope is typically discouraged since it's bad... very bad... probably the worst ever! And we're trying to make code great again!